.github/workflows/gh-ci-cron.yaml

```diff
@@ -32,29 +32,34 @@ jobs:
       with:
         os-type: "ubuntu"
 
-    - name: setup_miniconda
-      uses: conda-incubator/setup-miniconda@v2
+    - uses: conda-incubator/setup-miniconda@v2
       with:
-        python-version: 3.9
-        auto-update-conda: true
-        channel-priority: flexible
+        python-version: "3.10"
+        miniforge-variant: Mambaforge
+        miniforge-version: latest
+        channel-priority: strict
         channels: conda-forge, bioconda
-        mamba-version: "*"
         add-pip-as-python-dependency: true
         architecture: x64
 
-    - name: install_dev_versions
-      run: |
-        pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
-        pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple scipy
-
-    - name: install_other_deps
+    - name: install_deps
       uses: ./.github/actions/setup-deps
       with:
         mamba: true
-        numpy: ""
-        scipy: ""
+        full-deps: true
+
+      # overwrite installs by picking up nightly wheels
+    - name: nightly_wheels
+      run: |
+        pip install --pre -U -i https://pypi.anaconda.org/scipy-wheels-nightly/simple scipy numpy h5py matplotlib
```

Comment:
comment: there's a lot of  .\\\\agic hidden in these action files... I would have no idea where to look for these wheels.

Reply:
I think one of the next major tasks here will be to a) document the CI actions, b) clean them up (it's grown a lot beyond what I initially planned back at the start of last year when I squashed things into using re-usable actions).